integration/airflow/openlineage/airflow/extractors/base.py

```diff
@@ -55,10 +54,8 @@ def __repr__(self):
 
 
 class BaseExtractor(ABC, LoggingMixin):
```

Comment:
Since we will be providing a way to define custom extractors, would now be a good time to rename the legacy class `StepMetadata` to `TaskMetadata`?

Reply:
Good idea. Mind if I do it in separate PR prior to next release? I'd do bigger cleanup of intermediate data structures and extractor API handling then.

Reply:
https://github.com/OpenLineage/OpenLineage/issues/289

Reply:
Sounds great!